src/Stormpath/Util/Magic.php

```diff
@@ -34,8 +34,8 @@ public function __construct()
      * @param string $property Property name
      * @return mixed|null Property value if it exists, null if not
      */
-    public function __get($property) {
-
+    public function __get($property)
```

Comment:
Don't we need some more changes here? Just curious if the following call would work when setting/getting the custom data property: 

```php
$customData = $account->customData;
$customData->rank = "Captain";
```